testing/skia_gold_client/tool/e2e_test.dart

```diff
@@ -0,0 +1,132 @@
+import 'dart:io';
```

Comment:
nit: this and the other new file need an engine copyright header.

Reply:
Done!